src/shared/codelens/codeLensUtils.ts

```diff
@@ -252,16 +240,12 @@ export async function makeCSharpCodeLensProvider(): Promise<vscode.CodeLensProvi
 }
 
 export function makeTypescriptCodeLensProvider(): vscode.CodeLensProvider {
-    const logger = getLogger()
-
     return {
         provideCodeLenses: async (
             document: vscode.TextDocument,
             token: vscode.CancellationToken
         ): Promise<vscode.CodeLens[]> => {
             const handlers = await tsCodelens.getLambdaHandlerCandidates(document)
-            logger.debug('makeTypescriptCodeLensProvider handlers: %O', handlers)
```

Comment:
I'm all for removing code - but out of interest what's the motivation behind removing these log lines? Are they cluttering up a log file somewhere?

to put it another way - what's the cost to keeping them?

Reply:
Yes, they are very noisy (and frequent) and not really useful. I added them awhile ago but don't need them anymore.

Good catch though, I agree in general.